package.json

```diff
@@ -41,7 +41,8 @@
     },
     "dependencies": {
         "cross-spawn": "^7.0.2",
-        "enquirer": "^2.3.5"
+        "enquirer": "^2.3.5",
+        "semver": "^7.7.1"
```

Comment:
I prefer to keep the package as small as possible, so introducing this package may not be worth it, but I'm not against it. 😄

Reply:
I think it's better to include the `semver` package because semver versioning checking is complicated enough that we don't want to get it wrong.